src/api/test/functional/webui/proxy_mode_test.rb

```diff
@@ -13,7 +13,9 @@ def test_access_as_new_user
     visit '/home'
     assert_equal '/user/show/pico', page.current_path
     assert User.where(login: 'pico').exists?
-    User.find_by(login: 'pico').destroy
+    u=User.find_by(login: 'pico')
```

Comment:
it is not new, just incomplete to my view :) 

It tests if a user gets created under this condition. So it should also test that it is a valid user imho.